content/docs/howto/php.md

```diff
@@ -61,113 +64,352 @@ app, include the values below in your `composer.json` file:
 {{< code/copyable >}}
 {
   "require": {
-    "php": ">=7.1"
+    "php": ">=8.0"
   }
 }
 {{< /code/copyable >}}
 
 If your app has a `composer.lock` file, the buildpack will use
 the php version defined there.
 
+#### Deprecated: Using buildpack.yml
+
+Specifying the PHP version through `buildpack.yml` configuration will be
+deprecated in PHP language family buildpack v1.0.0.  To migrate from using
+`buildpack.yml` please set the `$BP_PHP_VERSION` environment variable.
+
+### Configure the PHP library directory
```

Comment:
Any sense of why someone might want to do this? A brief mention of a use-case could be helpful.